src/background/adblocker.js

```diff
@@ -92,21 +92,28 @@ async function updateEngines() {
 
     if (enabledEngines.length) {
       let updated = false;
+
       // Update engines from the list of enabled engines
-      for (const id of enabledEngines) {
-        if (id === engines.CUSTOM_ENGINE) continue;
-        updated = (await engines.update(id).catch(() => false)) || updated;
-      }
+      await Promise.all(
```

Comment:
What's the reason to do it in parallel ? Wasn't it fine to do it one after other?